course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -108,6 +110,9 @@
         };
         $scope.closeAlert = function(source) {
             $scope[source] = null;
+            if($scope.partialFailureData){
```

Comment:
nitpick: does this need to be in a conditional, or can we just always set it to `null`?